tekton/src/params/install.yml.j2

```diff
@@ -651,3 +651,126 @@
   type: string
   description: controls the workload size of predict containers
   default: "small"
+
```

Comment:
To make this install file more manageable we have started breaking up the params .... e.g. see the various `tekton/src/params/install-x.yml.j2` files:

AI Broker params should be introduced in a similar way:

```
# Dependencies - IBM Maximo AI Broker
# -----------------------------------------------------------------------------
{{ lookup('template', params_src_dir ~ '/install-aibroker.yml.j2') }}
```

and then have this content in `tekton/src/params/install-aibroker.yml.j2`

Reply:
this is fixed